sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractEditForm.java

```diff
@@ -503,6 +513,16 @@ protected void addNonPrimaryDiseasesTo(ComboBox diseaseField) {
 		}
 	}
 
+	protected void removeFollowUpDisabledDiseases(ComboBox diseaseField) {
```

Comment:
please merge this with the above function to avoid requesting and iterating over disease configurations

Reply:
this doesn't allow changing the followupEnabled from the DB because you always take the default value of the enum, you should use disease configuration instances to check wether followup is enabled or not.
So the filtering should be done on the backend

Reply:
update the method to retrieve the followUpEnabled diseases